src/config/chis-ke/config.json

```diff
@@ -266,7 +267,7 @@
           "friendly_name": "CHA Name",
           "property_name": "name",
           "type": "name",
-          "parameter": ["\\(PHO\\)", "\\(CHA\\)"],
+          "parameter": ["\\(\\s*PHO\\s*\\)", "\\(\\s*CHA\\s*\\)"],
```

Comment:
note this change was just during testing. i saw inputs like Richard Nixon (PHO ) which were not being properly formatted